smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftSettings.kt

```diff
@@ -36,6 +36,7 @@ private const val GIT_REPO = "gitRepo"
 private const val SWIFT_VERSION = "swiftVersion"
```

Comment:
The visibility setting in swift plugin codegen settings come from the smithy-build.json, like all other values in the settings.

It's added to generate internal STS client into AWSSDKIdentity module. For all public services, visibility is public.